src/js/utils/Select.jsx

```diff
@@ -262,7 +262,7 @@ class Select extends Component {
       multi = false, delimiter = ';', async = false, showValueTooltip, showLabelTooltip,
       clearable = true, arrowLeft, arrowUp, arrowRight, arrowDown, fieldRef, onTabPress,
       onEnterPress, customSelectComponents, optionRenderer, classNamePrefix,
-      showSelectedOptionColor, scrollableParentContainerClassName, ...attributes
+      showSelectedOptionColor, scrollableParentContainerClassName, onKeyDown, ...attributes
```

Comment:
why did you need to edit the old select?

Reply:
+1